### DISCUSSION TITLE
This project still alive?

### DISCUSSION THREAD

**Discussion by eitsupi** [devcontainers-community/.github#9] [Q&A] [answered]
@jcbhmr I feel that you continue to ignore mentions from me. (See https://github.com/devcontainers-community/features/pull/24#pullrequestreview-1510404298, https://github.com/devcontainers-community/features/pull/24#issuecomment-1652678423)
Are you still willing to continue this project?

If you are not willing to continue this, it may be worth stating in the README that this project has been abandoned. (I may do that if I don't hear back from you)

**Comment by eitsupi** [comment]
@jcbhmr Could you take a look at this?

**Reply by eitsupi** [reply]
@jcbhmr Friendly ping

**Comment by jcbhmr** [comment, marked as answer]
> @jcbhmr I feel that you continue to ignore mentions from me. (See [devcontainers-community/features#24 (review)](https://github.com/devcontainers-community/features/pull/24#pullrequestreview-1510404298), [devcontainers-community/features#24 (comment)](https://github.com/devcontainers-community/features/pull/24#issuecomment-1652678423))
> Are you still willing to continue this project?

You're absolutely right. I have. 😢 No excuse, just general lack of motivation/interest in this area for a while. I'll be honest and say I felt guilty for completely ignoring everything 😬

As for continuing the project...

I've recently had a resurgence of interest/inspiration/ideas. I think this is enough to fill my "energy bar" again. I think it also comes down to personal usage. A few months ago I used devcontainers and Codespaces _really_ heavily for university. Now that it's the summer, not so much. 

As for longer term: I'd like it to outlast me. Ideally I'd like to add more people like these fine fellows who are making devcontainer features but aren't in the "official index" to this project since they seem like they're making cool features. like https://github.com/FoodieAppServices/vsc-features and https://github.com/rocapp/features

So in essence: It's kinda on-a-whim and I haven't had the whim in a bit.

**Reply by eitsupi** [reply]
Ok, thanks for the reply. Glad to see you back.

I am leaving this organization because I do not want to be involved in any more of your whims.

**Reply by jcbhmr** [reply]
Sounds good. 👍 Happy devcontainer-ing! 🚀
